### PR TITLE
Implemented insert, update and delete queries for carrier service table

### LINKED ISSUE #8016: [Shopify] Implement INSERT/UPDATE/DELETE for the CarrierService table
MindsDB integrates with Shopify, so you can fetch data from your online store and use it to make relevant predictions and forecasts with MindsDB.

## Resources

Familiarize yourself with the existing Shopify integration: https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/shopify_handler

## Steps to Follow

The task is to extend [this implementation](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/shopify_handler) to include the INSERT, UPDATE, and DELETE commands for the [CarrierService table](https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/shopify_handler/shopify_tables.py#L603).

Please provide screenshots or a short video presenting this feature.

## :loudspeaker: Additional rewards :trophy:

Each integration extension brings 15+ points for SWAG and entry into the draw for a :point_down: :
-  :computer:  [Razer Blade 16 Laptop](https://www.razer.com/gaming-laptops/Razer-Blade-16/RZ09-0483SEJ3-R3U1)
- :shirt: :bear:  [MindsDB Swag](https://store.covver.io/mindsdb/collections/swag-collection)

:information_source: For more info check out https://mindsdb.com/hacktoberfest/ :point_left:

### PR DESCRIPTION
**Pull request by Om-Doiphode**
## Description

Please include a summary of the change and the issue it solves. 
Implemented Insert, Update, and Delete queries for the Carrier Service Table. 

Fixes #8016 

## Type of change

(Please delete options that are not relevant)

- [ ] ⚡ New feature (non-breaking change which adds functionality)

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

Insert query: 

Screenshot from 2023-10-24 10-07-44

Screenshot from 2023-10-24 10-08-07

Delete query: 

Screenshot from 2023-10-24 10-14-11

Screenshot from 2023-10-24 10-15-19

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by Om-Doiphode** [comment]
I am facing an issue with the update query. 
image
On using the above query I am getting an error. I tried debugging by printing the values for `values_to_update`, `where_conditions`, `carrier_services_df` in the terminal. 
image
As can be seen in the above image, the entry is present in the table still it gives the above error. Can you please look into this?

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]
Looks good, let's keep update and delete in this PR and remove the insert. Once you solve the issues you can open another PR with the insert?

**Comment by Om-Doiphode** [comment]
recheck

**Comment by Om-Doiphode** [comment]
> Looks good, let's keep update and delete in this PR and remove the insert. Once you solve the issues you can open another PR with the insert?

The insert and delete queries are working fine, there seems to be an issue with the update query. Should I keep only the insert and delete queries then? I will try to solve the issue related to update query and will open a new PR for the update query. Does this work?

**Comment by someshfengde** [comment]
Hi @ZoranPandovski I'm also facing the same error while updating the database.

**Comment by supunvindula** [comment]
hi @Om-Doiphode , can you guide me on testing these implementations? Thank you!

**Comment by Om-Doiphode** [comment]
Hi @ZoranPandovski, is there anything else that needs to be done?

File: mindsdb/integrations/handlers/shopify_handler/shopify_tables.py [hunk 1]
```diff
@@ -641,6 +641,101 @@ def select(self, query: ast.Select) -> pd.DataFrame:
...
+        Raises
+        ------
+        ValueError
+            If the query contains an unsupported condition
+        """
+        update_statement_parser = UPDATEQueryParser(query)
+        values_to_update, where_conditions = update_statement_parser.parse_query()
+
+        print(f"values_to_update: {values_to_update}")
+        print(f"where_conditions: {where_conditions}")
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/shopify_handler/shopify_tables.py:726]: Please remove the print

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]
Also please merge the latest staging before we merge this

**Comment by Om-Doiphode** [comment]
Hi @ZoranPandovski, I have made the suggested changes, please review. Thanks!

**Comment by Om-Doiphode** [comment]
Hi @ZoranPandovski, I have merged the latest staging also.

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**